Example/Tests/ApiTests.m

```diff
@@ -91,10 +108,29 @@
         b = block;
         done();
       }];
+      NSError *e;
+      [Textile.instance.node stop:&e];
+      expect(e).beNil();
     });
     assertWithTimeout(60, thatEventually(@([delegate.updatedItems containsObject:b.id_p])), isTrue());
-    assertWithTimeout(60, thatEventually(@([delegate.completeItems containsObject:b.id_p])), isTrue());
```

Comment:
I removed this check because I don't think it's possible to guarantee these items are processed by the OS at the node level. We can however ensure they are written to disk and updated.